src/_blog/rust-libp2p-browser-to-server.md

```diff
@@ -0,0 +1,87 @@
+---
+tags:
+  - rust-libp2p
+  - rust
+  - libp2p
+  - browser
+  - webrtc
+  - webassembly
+  - wasm
+title: rust-libp2p in the Browser with WebRTC!
+description: News about how rust-libp2p in the browser connects to nodes in a network using WebRTC
+date: 2023-12-07
+permalink: ''
+translationKey: ''
+header_image: /rust-libp2p.jpeg
+author: DougAnderson444
+---
+
+# Rust-libp2p WebRTC now available in the Browser
+
+We are excited to announce that [rust-libp2p](https://github.com/libp2p/rust-libp2p) running in the browser can now establish [WebRTC connections](https://webrtc.org/). Before now, WebRTC was only available on the server in [rust-libp2p](https://github.com/libp2p/rust-libp2p), but after months of coding and reviews, we are pleased to annouce it is available on both the server and in the browser from rust-libp2p.
+
+# Why WebRTC for rust-libp2p?
+
+There are a few different Transports to choose from, each with their pros and cons. So, why WebRTC? Let's dive into some of the benefits of WebRTC and why we decided it was important to implement as a Transport within rust-libp2p for use in the browser.
```

Comment:
```suggestion
There are a few different options for a `Transport` to choose from, each with its own pros and cons. So, why WebRTC? Let's dive into some of the benefits of WebRTC and why we decided it was important to implement it as a `Transport` within rust-libp2p for use in the browser.
```

Reply:
Good idea, I will use the code block with the `T`